eng/common/templates/job/publish-build-assets.yml

```diff
@@ -53,10 +53,17 @@ jobs:
         downloadPath: '$(Build.StagingDirectory)/Download'
       condition: ${{ parameters.condition }}
       continueOnError: ${{ parameters.continueOnError }}
-    
+
     - ${{ if and(eq(parameters.runAsPublic, 'false'), ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
       - task: NuGetAuthenticate@0
 
+    - task: PowerShell@2
```

Comment:
My understanding has typically been that anything under `eng/common` is "stuff arcade owns". Will this be overridden if arcade bumps up a version?

Presumably the fix would be in far before we even do that, but I'm curious.

Reply:
This fix will block arcade update till I get the other fix out. I am going to be working on it next week. I will have to coordinate with the arcade-services roll out. I will keep you posted on that.